### ISSUE TITLE
Using integer keys causes get/set to not work as expected

### ISSUE THREAD

**Issue by okhwaja** [brigade/mock_redis#157]
Hi,

I'm running into an issue when using a `SET` followed by a `GET` with integer keys. My production`redis-rb` client handles this case.

```
[1] pry(main)> r = Redis.new;
[2] pry(main)> r.set(123, 'yo')
  Redis (3.30ms) SET 123 yo
=> "OK"
[3] pry(main)> r.get(123)
  Redis (0.65ms) GET 123
=> "yo"
[4] pry(main)> m = MockRedis.new;
[5] pry(main)> m.set(123, 'yo')
=> "OK"
[6] pry(main)> m.get(123)
=> nil
```

The key is converted into a [string in the `SET` method](https://github.com/brigade/mock_redis/blob/master/lib/mock_redis/string_methods.rb#L201), but [not in the `GET` method](https://github.com/brigade/mock_redis/blob/master/lib/mock_redis/string_methods.rb#L87)

I'd be happy to submit a PR to change this behavior. Please let me know

**Comment by sds** [comment]
Thanks for the report, @okhwaja. Would love a pull request to make this consistent. Should be easy to add a test for, as well.

**Comment by sds** [comment]
Fixed in #158. Thanks!
